openid-4-verifiable-presentations-1_0.md

```diff
@@ -611,6 +630,29 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Verifier Attestations {#verifier-attestations}
+
+Verifier Attestations allow the Verifier to provide additional context or metadata as part of the Authorization Request. These inputs can support a variety of use cases, such as helping the Wallet apply policy decisions, validating eligibility, or presenting more meaningful information to the End-User during consent.
+
+Each Verifier Attestation is an object containing a type identifier, associated data and optionally references to credential ids. The format and semantics of these attestations are defined by ecosystems or profiles.
+
+For example, a Verifier might include:
+
+- A **registration certificate** issued by a trusted authority, to prove that it is authorized to request certain credentials.
+- An **attestation of intended use**, declaring why a particular credential is being requested.
```

Comment:
difference to first bullet?

Reply:
It's not a clear different. I would remove the first point since the second one is more generic.